src/ui/component/field/field.tsx

```diff
@@ -6,7 +6,7 @@ import './field.scss'
 type FieldProps = {
   id: string
   label?: string
-  onChange: (value: string) => void
+  onChange?: (value: string) => void
```

Comment:
I think `onChange` should never be optional in a input component

Reply:
What about in cases when the field is `readonly`? That's why I changed it... :/

Reply:
optional is good :)